src/lib/BridgeBase.sol

```diff
@@ -182,8 +183,7 @@ abstract contract BridgeBase is OwnableUpgradeable, UUPSUpgradeable {
         SharedStructs.ContractStateHash[] memory hashes = new SharedStructs.ContractStateHash[](tokenAddresses.length);
 
         if (!shouldFinalizeEpoch()) {
-            // console.log("No need to finalize");
-            return;
+            revert NoStateToFinalize();
```

Comment:
Do you want this to revert?

Reply:
Yeah, it is done on purpose for the relayers